test/Microsoft.AspNet.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -288,7 +292,7 @@ public static IEnumerable<object[]> RazorViewEngine_UsesExpandersForLayoutsData
 View With Layout
 </language-layout>";
 
-                yield return new[] { "gb", expected1 };
+                yield return new[] { "en-GB", expected1 };
```

Comment:
Well should it be `CultureInfo.TwoLetterISOLanguageName` vs `CultureInfo.Name`? `.Name` generally refers to a locale, while `.TwoLetterISOLanguageName` refers to a language